src/routes/routes.ts

```diff
@@ -369,6 +374,48 @@ function register(app: express.Application) {
     etapiSpecRoute.register(router);
     etapiBackupRoute.register(router);
 
+    // Embeddings API endpoints
+    apiRoute(GET, "/api/embeddings/similar/:noteId", embeddingsRoute.findSimilarNotes);
+    apiRoute(PST, "/api/embeddings/search", embeddingsRoute.searchByText);
+    apiRoute(GET, "/api/embeddings/providers", embeddingsRoute.getProviders);
+    apiRoute(PATCH, "/api/embeddings/providers/:providerId", embeddingsRoute.updateProvider);
+    apiRoute(PST, "/api/embeddings/reprocess", embeddingsRoute.reprocessAllNotes);
+    apiRoute(GET, "/api/embeddings/queue-status", embeddingsRoute.getQueueStatus);
+    apiRoute(GET, "/api/embeddings/stats", embeddingsRoute.getEmbeddingStats);
+    apiRoute(GET, "/api/embeddings/failed", embeddingsRoute.getFailedNotes);
+    apiRoute(PST, "/api/embeddings/retry/:noteId", embeddingsRoute.retryFailedNote);
+    apiRoute(PST, "/api/embeddings/retry-all-failed", embeddingsRoute.retryAllFailedNotes);
+    apiRoute(PST, "/api/embeddings/rebuild-index", embeddingsRoute.rebuildIndex);
+    apiRoute(GET, "/api/embeddings/index-rebuild-status", embeddingsRoute.getIndexRebuildStatus);
+
+    // LLM chat session management endpoints
+    apiRoute(PST, "/api/llm/sessions", llmRoute.createSession);
```

Comment:
aren't these missing the CSRF/Auth middleware – or am I missing something

Reply:
Some of them sure were lol - migrated them from `route()` to `apiRoute()`